### ISSUE TITLE
Why does DeepClean take so long to get to an ASD of 1?

### ISSUE THREAD

**Issue by alecgunny** [ML4GW/DeepClean#25]
Inspection of a [typical DeepClean loss curve](https://ml4gw.github.io/DeepClean/) shows that the training ASDR takes nearly 100 epochs to get to an ASDR of 1, where it hovers for a while until the learning rate is dropped (due to going too long without a validation loss improvement), at which point it quickly drops down to the mid-0.8s where it eventually lands.

Why is it so hard for DeepClean to arrive at an ASDR of 1.0? This should be a pretty trivial solution to arrive at: just output all 0s. Could training be accelerated if the weight and bias of the last convolutional layer were initialized to 0, and the learning rate was ramped up from some initially really small value to keep DeepClean from deviating too far from this solution without a very good reason for doing so?

It's possible that the learning rate is just too high from the start, but some very hand-wavy hyperparameter searching has indicated this doesn't seem to be the case, though this should obviously be verified more rigorously.

**Comment by alecgunny** [comment]
This is no longer an issue, not sure what was causing the problem unfortunately
